docs/network/index.html

```diff
@@ -1323,6 +1323,24 @@ <h2 id="Events">Events</h2>
   }
 }
 </pre>
+
+This is the structure common to all events. Specifically for the click event, the following property is added:
+
+<pre class="prettyprint lang-js">{
+...
+  items: [Array of click items],
+}</pre>
+
+Where the click items can be:
+<pre class="prettyprint lang-js">
+  {nodeId:number}            // node with given id clicked on
+  {nodeId:number labelId:0}  // label of node with given id clicked on
+  {edgeId:number}            // edge with given id clicked on
+  {edge:number, labelId:0}   // label of edge with given id clicked on
+</pre>
+
+The order of the <code>items</code> array is descending in z-order.
+Thus, to get the topmost item, get the value at index 0.
                 </td>
```

Comment:
Does this change require a follow up pr to the gh-pages branch?
If so, can you create an issue and reference it from this pr?

Reply:
I don't think so; the docs in `gh-pages` are updated from `develop` upon release.